src/executables/file_hash.c

```diff
@@ -215,7 +216,9 @@ fh_create (const char *name, int est_size, int page_size, int cached_pages, cons
   /* Open the hash file */
   if (!hash_filename || hash_filename[0] == '\0')
     {
-      ht->hash_filename = strdup (tmpnam (NULL));
+      auto[filename, filedes] = filesys::open_temp_filedes ("fhash_");
```

Comment:
Personally, it would be better to use the same prefix for all temporary files created in CUBRID. However, the opinion of @mhoh3963  is the most important. I will follow him.

PS. Is there any reason to use a different prefix like "fhash_"?

Reply:
I think the only reason is to better differentiate the temp files between them, to be easier to find, in case that is necessary.